libs/accounts/src/lib/transfer-form.spec.tsx

```diff
@@ -27,6 +33,34 @@ describe('TransferForm', () => {
     submitTransfer: jest.fn(),
   };
 
+  it('validates a manually entered address', async () => {
+    render(<TransferForm {...props} />);
+    submit();
+    expect(await screen.findAllByText('Required')).toHaveLength(3);
+    const toggle = screen.getByText('Enter manually');
+    fireEvent.click(toggle);
+    // has switched to input
+    expect(toggle).toHaveTextContent('Select from wallet');
+    expect(screen.getByLabelText('Vega key')).toHaveAttribute('type', 'text');
+    fireEvent.change(screen.getByLabelText('Vega key'), {
+      target: { value: 'invalid-address' },
+    });
+    await waitFor(() => {
+      const errors = screen.getAllByTestId('input-error-text');
```

Comment:
suggestion (non blocking): this can be made more succinct (and no nesting with `const errors = await screen.findAllByTestId(...)`